opentelemetry-otlp/tests/integration_test/tests/logs.rs

```diff
@@ -99,6 +99,7 @@ mod logtests {
         Ok(())
     }
 
+    #[ignore = "TODO: [Fix Me] Ignored due to concurrent test failures. Needs resolution."]
```

Comment:
> Running these tests in single thread using --test-thread=1 doesn't seem to help.

If running the tests sequentially didn't help, then concurrency is not the issue, right?

```suggestion
    #[ignore = "TODO: [Fix Me] Failing on CI. Needs to be investigated and resolved."]
```